data/brands/amenity/bank.json

```
@@ -7130,15 +7130,15 @@
       }
     },
     {
-      "displayName": "Takarékszövetkezet",
+      "displayName": "Takarékbank",
       "id": "takarekszovetkezet-60884a",
       "locationSet": {"include": ["hu"]},
```

This will also capture the old name too, if every Takarékszövetkezet became a Takarékbank?

```suggestion
      "locationSet": {"include": ["hu"]},
      "matchNames": [
        "Takarékszövetkezet"
      ],
```

---

Yes